### ISSUE TITLE
Alienware 13 R1 Fixes

### ISSUE THREAD

**Issue by buzyn** [rsm-gh/akbl#63]
# System information

+ Distribution name: Antegros
+ Distribution version: 5.1.15
+ Python3 version: `python3 --3.7.3-2`
+ akbl version:  akbl-v2019.05.26
+ Computer model: Alienware 13 (r1)
+ Daemon status: `akbl --daemon-is-on`

# Bug description
good day. 
recently replaced the battery and ssd drive on my alienware 13 (r1)
installed windows10, program drivers, respectively, installed antegros + kde.
after all the installations I decided, of course, to adjust the backlight, I was immediately upset by the blue color of the logo.
The keyboard highlights normally with all colors. but the blue backlight is on the outer lid, alienware logo, shining at half the power.
From this, when choosing a white color, the logo shines yellow. When choosing a blue color - the logo shines green.
help please understand.
Is this a controller problem, or can it be programmatically adjusted?
Is it possible to somehow raise the voltage to blue in a software way?

**Comment by tuxkernel** [comment]
Maybe #53

**Comment by rsm-gh** [comment]
Hello @buzyn ,

I completely ignore what could be causing your problem. Since some time ago, some computers needed to change the variable `DATA_LENGTH ` from `9` to `12`, which caused color problems (because the data is not formatted as it should for the hardware).

I checked your computer info and it has the `DATA_LENGTH=9` same as mine, and my colors work good. Since I'm not capable of reproducing the bug I wont be able to fix it.

Regards,
rsm

**Comment by rsm-gh** [comment]
I'm going to close this issue since the color problems will be treated [here](https://github.com/rsm-gh/akbl/issues/65).

**Comment by buzyn** [comment]
good day.
I still restored the backlight. but it was not a software solution.
I replaced the rgb led modules in the display cover.
I have 2 identical laptops. I replaced the entire display cover.
identified the problem in this way.
Accordingly, the problem is not in the program, but in the LEDs.
Perhaps they could not stand the load when using white backlight. I believe that the manufacturer did not specifically use white in its gamut. the LED is very small and it may have overheated.
By the way, all the LEDs in the display lid stopped working at full power in blue.
